cfme/infrastructure/pxe.py

```diff
@@ -644,29 +648,12 @@ class SystemImageType(Updateable, Pretty, Navigatable):
     VM_OR_INSTANCE = "VM and Instance"
     HOST_OR_NODE = "Host / Node"
 
-    def __init__(self, name=None, provision_type=None, appliance=None):
-        Navigatable.__init__(self, appliance=appliance)
-        self.name = name
-        self.provision_type = provision_type
-
-    def create(self, cancel=False):
-        """
-        Creates a System Image Type object
-
-        Args:
-            cancel (boolean): Whether to cancel out of the creation.  The cancel is done
-                after all the information present in the SIT has been filled in the UI.
-        """
-        view = navigate_to(self, 'Add')
-        view.fill({'name': self.name, 'type': self.provision_type})
-        main_view = self.create_view(PXESystemImageTypesView)
-        if cancel:
-            view.cancel.click()
-            msg = 'Add of new System Image Type was cancelled by the user'
-        else:
-            view.add.click()
-            msg = 'System Image Type "{}" was added'.format(self.name)
-        main_view.flash.assert_success_message(msg)
+    # def __init__(self, name=None, provision_type=None, appliance=None):
```

Comment:
**REQUIRED:** Please remove these comment lines

Reply:
I missed that, thanks for highlighting 👍